### PR TITLE
refactor smartContract tests to use async/await

### PR DESCRIPTION
**Pull request by b-pmcg**

### CODE REVIEW

File: test/eth/SmartContractService.spec.js [hunk 1]
```diff
@@ -15,65 +15,48 @@ test('getContractByName should have proper error checking', done => {
     'Cannot resolve network ID. Are you connected?'
   );
 
-  service
-    .manager()
-    .authenticate()
-    .then(() => {
-      expect(() =>
-        service.getContractByName(contracts.SAI_TOP, { version: 999 })
-      ).toThrow('Cannot find contract SAI_TOP, version 999');
```
Comment (levity) [inline test/eth/SmartContractService.spec.js]: Keeping `expect ... toThrow` here is a bit more idiomatic Jest

Comment (b-pmcg) [reply test/eth/SmartContractService.spec.js]: Nice, I didn't know you could do it this way with async functions but its much cleaner.

Comment (levity) [reply test/eth/SmartContractService.spec.js]: `getContractByName` isn't async :)

Comment (b-pmcg) [reply test/eth/SmartContractService.spec.js]: oops, totally whiffed on that one 🙄

**Pull request merged**